usaspending_api/etl/management/commands/create_delta_table.py

```diff
@@ -48,6 +48,11 @@ def add_arguments(self, parser):
             help="An alternate delta table name for the created table, overriding the TABLE_SPEC destination_table "
             "name",
         )
+        parser.add_argument(
```

Comment:
To make sure I follow, we would want to create as part of the deploy some data change steps that would first recreate the CDF tables and reload them with CDF enabled?